server/src/main/java/com/objectcomputing/checkins/services/memberprofile/retentionreport/RetentionReportServicesImpl.java

```diff
@@ -163,12 +137,10 @@ private RetentionReportResponseDTO getReportValues(LocalDate periodStartDate, Lo
 
     public float getTurnoverRate(LocalDate dateToStudy, List<MemberProfile> memberProfiles, FrequencyType frequency) {
         LocalDate beginningDate;
-        if (frequency == FrequencyType.DAILY) {
-            beginningDate = dateToStudy.minusDays(1);
-        } else if (frequency == FrequencyType.WEEKLY) {
-            beginningDate = dateToStudy.minusMonths(1);
-        } else {
-            beginningDate = dateToStudy.minusMonths(1);

```

Comment:
WEEKLY is `dateToStudy.minusMonths(1)`

But so is the default (`MONTHLY`)... is this right?

Reply:
Yeah, not going to lie...I don't think this bit of logic is right. I think some wires got crossed on this bit of fuctionality. I need to go back and find the original task and create some bug fix stories. Luckily this bit doesn't have a UI yet.